src/test/java/words/CounterTest.java

```diff
@@ -0,0 +1,65 @@
+package words;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.Map;
+
+public class CounterTest {
+    private Counter counter;
+
+    @Before
+    public void set_up(){
+        counter = new Counter();
+    }
+
+    @Test
+    public void count_word(){
+        Assert.assertTrue(counter.count("Hello"));
+        Assert.assertFalse(counter.count(null));
+        Assert.assertFalse(counter.count(""));
+        Assert.assertFalse(counter.count(" "));
+        Assert.assertFalse(counter.count("   "));
+    }
+
+    @Test
+    public void frequency(){
```

Comment:
I will recommend to simplify test. 
You can call several times count
e.g. `count('Hello world")`, `count("Hello Vetal")`, `count("a")`
```
Map<String, Integer> expected = new HashMap<>()
expected.put("hello", 2)
expected.put("world", 1)
...
assertEquals(expected, counter.getFrequency)
```